src/main/java/uk/gov/ons/census/notifyprocessor/service/EnrichedFulfilmentRequestService.java

```diff
@@ -30,6 +33,7 @@ public void processMessage(EnrichedFulfilmentRequest fulfilmentRequest) {
           UUID.randomUUID().toString(),
           senderId);
     } catch (NotificationClientException e) {
+      log.with("exception", e).error("Gov Notify sendSms error");
```

Comment:
log details of failed fulfilmentRequest, at the very least templateId

Reply:
we can inspect the entire message using the queuetool. The only reason why we need to log anything here is because we are converting from a checked exception to a runtime exception, and we lose detail even though we bundle the original exception into the runtime exception. What we care about here is any clue as to why Gov Notify rejected the message, not what the contents of _our_ message is, which can already be easily inspected.